packages/reports/src/Containers/open-positions.jsx

```diff
@@ -505,31 +553,70 @@ const OpenPositions = ({
             {active_positions.length !== 0 && (
                 <React.Fragment>
                     <DesktopWrapper>
-                        <div className='open-positions__contract-types-selector__container'>
-                            <Dropdown
-                                is_align_text_left
-                                name='contract_types'
-                                list={contract_types}
-                                value={contract_type_value}
-                                onChange={handleChange}
-                            />
+                        <div
+                            className={
+                                is_accumulator_selected
+                                    ? 'open-positions__accumulator-container'
+                                    : 'open-positions__contract-types-selector-container'
+                            }
+                        >
+                            <div className='open-positions__accumulator-container__contract-dropdown'>
+                                <Dropdown
+                                    is_align_text_left
+                                    name='contract_types'
+                                    list={contract_types}
+                                    value={contract_type_value}
+                                    onChange={handleContractChange}
+                                />
+                            </div>
+
+                            {is_accumulator_selected && (
+                                <div className='open-positions__accumulator-container__rates-dropdown'>
+                                    <Dropdown
+                                        is_align_text_left
+                                        name='accumulator_rates'
+                                        list={accumulator_rates}
```

Comment:
related to another comment: https://github.com/maryia-deriv/deriv-app/pull/86/files#r1106162428
```suggestion
                                        list={accumulators_rates_list}
```